src/update/UpdateNoticeDialog.cpp

```diff
@@ -117,12 +118,19 @@ UpdateNoticeDialog::UpdateNoticeDialog(wxWindow* parent)
 
    S.EndVerticalLay();
 
-   Layout();
    Fit();
+   Layout();
```

Comment:
What is the effect of swapping those lines?  It's Layout, then Fit almost everywhere else.